src/Microsoft.TestPlatform.ObjectModel/TestProcessAttachDebuggerPayload.cs

```diff
@@ -27,4 +25,22 @@ public TestProcessAttachDebuggerPayload(int pid)
     /// </summary>
     [DataMember]
     public int ProcessID { get; set; }
+
+    [DataMember]
+    // Added in version 7.
```

Comment:
Would the use of `ProtocolVersion` attribute make sense here ?

Reply:
Yes, I commented the same thing above. I just did not want to fiddle with usings, and adding too many overloads for that attribute, I will do that later.